documentation/content/howtocontribute.rst

```diff
@@ -157,3 +157,42 @@ If you like, you can put extra tags at the end:
 You can also create your own tag or use one that’s in common usage such as
 `Co-authored-by` or `Helped-by`.
 
+About Issues Labels
+-------------------
+We use Github issues and milestones as our tool for managing kw project. We use
+issues for keeping track of different things, as a result, we categorize all of
+them with labels; most of the available labels are self-explanatory. However,
+few of them deserver a brief explanation:
+
+* **done: wait for stable**: In the section *Development Cycle and Branches* we
+  explained how our branch management works, recall that we have an unstable
+  branch that soon or later will be applied to the master branch (stable). This
+  label is used for identifying issues already solved on the unstable, but not
+  landed in the master yet. If you see this label, feel free to test the
+  feature/bugfix in the unstable branch.
+* **new feature request**: If you want to request or see a possible new
+  feature, you should use this label.
+* **good first issue**: If you started to use kw, and after that, you feel
+  determined to contribute to it but you don't know where you could start, this
```

Comment:
Maybe "feel determined to contribute" could be swapped by "desire to contribute"? (Just to make it a little more appealing for newcomming contributors)